example/advection/advection_package.cpp

```diff
@@ -270,22 +267,21 @@ Real EstimateTimestep(std::shared_ptr<Container<Real>> &rc) {
   IndexRange jb = pmb->cellbounds.GetBoundsJ(IndexDomain::interior);
   IndexRange kb = pmb->cellbounds.GetBoundsK(IndexDomain::interior);
 
-  Real min_dt = std::numeric_limits<Real>::max();
   auto &coords = pmb->coords;
 
   // this is obviously overkill for this constant velocity problem
-  for (int k = kb.s; k <= kb.e; k++) {
-    for (int j = jb.s; j <= jb.e; j++) {
-      for (int i = ib.s; i <= ib.e; i++) {
+  Real min_dt;
+  pmb->par_for(
```

Comment:
if you were to try the simd loop pattern, it would fail at compile time.  In the `MeshBlock` wrappers, this is already handled transparently.  In fact, I've been testing this in an environment where the DEFAULT_LOOP_POLICY is the simd for loop, and the code automatically switches to the MDRange loop if you're doing a reduction.

Reply:
Oh nice. Okay I missed that somewhere. My above comment about making a default reduce pattern still holds though, I think. As the most efficient pattern for reduction may not be the same as the most efficient pattern for a stencil.

Reply:
I started going down that path, but abandoned it (probably mostly out of laziness).  This solution works for now, so I figured we could return to this if we find reductions showing up performance wise.

Reply:
Yeah fair enough.